javascript/index.js

```diff
@@ -1,30 +1,57 @@
 const chronometer = new Chronometer();
-
 // get the buttons:
+
 const btnLeft = document.getElementById('btnLeft');
-const btnRight = document.getElementById('btnRight');
+btnLeft.onclick = function () {
+  if (btnLeft.className === "btn start") {
+    chronometer.startClick(printTime);
+    // printTime as a callback
+    // two functions printMinutes and printSeconds
+  } else if (btnLeft.className === "btn stop") {
+    chronometer.stopClick()
+  }
+};
+const btnRight = document.getElementById("btnRight");
+btnRight.onclick = function () {
+  if (btnRight.className === "btn reset") {
+    chronometer.resetClick()
+  } else if (btnRight.className === "btn split") {
+    chronometer.splitClick()
+  }
+}
+
+// const splitTime = document.getElementById("splits")
+// btnRight.onclick = function () {
+//   return chronometer.splitClick()
+// }`
 
 // get the DOM elements that will serve us to display the time:
-let minDec = document.getElementById('minDec');
-let minUni = document.getElementById('minUni');
-let secDec = document.getElementById('secDec');
-let secUni = document.getElementById('secUni');
-let milDec = document.getElementById('milDec');
-let milUni = document.getElementById('milUni');
-let splits = document.getElementById('splits');
+// let minDec = document.getElementById('minDec');
+// let minUni = document.getElementById('minUni');
+// let secDec = document.getElementById('secDec');
+// let secUni = document.getElementById('secUni');
+// let milDec = document.getElementById('milDec');
+// let milUni = document.getElementById('milUni');
+// let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  printMinutes();
+  printSeconds();
+  printMilliseconds();
 }
 
 function printMinutes() {
-  // ... your code goes here
+  minUni.innerHTML = chronometer.twoDigitsNumber(chronometer.getMinutes())[1];
+  minDec.innerHTML = chronometer.twoDigitsNumber(chronometer.getMinutes())[0];
 }
 
 function printSeconds() {
-  // ... your code goes here
+  secUni.innerHTML = chronometer.twoDigitsNumber(chronometer.getSeconds())[1];
+  secDec.innerHTML = chronometer.twoDigitsNumber(chronometer.getSeconds())[0];
 }
 
+
+
```

Comment:
For clearing the splits, what you want is basically to access the same element with a "splits" identifier and remove any list item by doing some DOM manipulation again
```
function clearSplits() {
  splits.innerHTML = '';
}